resources/assets/components/pages/AccountPage/VoterReg/VoterRegStatusBlock.js

```diff
@@ -1,47 +1,99 @@
 import React from 'react';
+import gql from 'graphql-tag';
 import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { useQuery } from '@apollo/react-hooks';
 
 import checkmark from './checkmark.svg';
-import rejectedIcon from './rejectedIcon.svg';
-import pendingIcon from './pendingIcon.svg';
+import Spinner from '../../../artifacts/Spinner/Spinner';
+import ErrorBlock from '../../../blocks/ErrorBlock/ErrorBlock';
+import { getTrackingSource } from '../../../../helpers/voter-registration';
 
 import './voter-reg.scss';
 
-const VoterRegStatusBlock = ({ status }) => {
-  if (status === 'PENDING') {
+const USER_VOTER_REGISTRATION_STATUS_QUERY = gql`
+  query userVoterRegistrationStatusQuery($userId: String!) {
+    user(id: $userId) {
+      id
+      voterRegistrationStatus
+    }
+  }
+`;
+
+const VoterRegStatusBlock = ({ userId }) => {
+  const options = { variables: { userId } };
+
+  const { loading, error, data } = useQuery(
+    USER_VOTER_REGISTRATION_STATUS_QUERY,
+    options,
+  );
```

Comment:
Since `options` is only used here, what do you think about just defining it here? I think it'd help with readability.
```suggestion
  const { loading, error, data } = useQuery(
    USER_VOTER_REGISTRATION_STATUS_QUERY,
     { variables: { userId } },
  );
```